types/chrome/test/index.ts

```diff
@@ -1150,7 +1150,10 @@ function testContextMenusCreate() {
         parentId: 1,
         type: 'normal',
         visible: true
-    }, () => console.log('created'));
+    };
+    chrome.contextMenus.create(creationOptions, () => console.log('created')); // $ExpectType void
+    chrome.contextMenus.create({ ...creationOptions, contexts: ['action', 'page_action'] }); // $ExpectType void
```

Comment:
I forgot to add a test for a context string, instead of an array of them:
```suggestion
    chrome.contextMenus.create({ ...creationOptions, contexts: ['action', 'page_action'] }); // $ExpectType void
    chrome.contextMenus.create({ ...creationOptions, contexts: 'page_action' }); // $ExpectType void
```